src/Maestro/SubscriptionActorService/PullRequestActor.cs

```diff
@@ -823,7 +825,48 @@ private async Task<string> CreatePullRequestAsync(List<UpdateAssetsParameters> u
                 var message = new StringBuilder();
                 await CalculateCommitMessage(coherencyUpdate.update, coherencyUpdate.deps, message);
                 await CalculatePRDescription(coherencyUpdate.update, coherencyUpdate.deps, description);
-                await darc.CommitUpdatesAsync(targetRepository, newBranchName, coherencyUpdate.deps.Select(du => du.To).ToList(), message.ToString());
+
+                List<GitFile> commitedFiles = await darc.CommitUpdatesAsync(targetRepository, newBranchName, coherencyUpdate.deps.Select(du => du.To).ToList(), message.ToString());
+
+                UpdatePRDescriptionDueConfigFiles(commitedFiles, description);
+            }
+        }
+
+        private void UpdatePRDescriptionDueConfigFiles(List<GitFile> commitedFiles, StringBuilder description)
+        {
+            GitFile globalJsonFile = commitedFiles.
+                Where(gf => gf.FilePath.Equals("global.json", StringComparison.OrdinalIgnoreCase)).
```

Comment:
What's the output for an update to the msbuild sdks section? Just an empty header?

Reply:
Oh nvm, you write the header if there's an SDK update. Maybe we should change it from "Updates to global.json" to Updates to dotnet SDKs used?

Reply:
Yeah, that's it. I believe the updates to msbuild sdks will also show up in the "Updates:" section as a package update?

Yeah, I can definitely change the title.

Reply:
Yeah, especially in the topic of shortening PR descriptions, I don't think it makes a lot of sense to include it in both places.